bke-development/gp-cicd-tools/values.yaml

```diff
@@ -129,6 +129,51 @@ argo_workflows:
       spec:
         ttlStrategy:
             secondsAfterCompletion: 3600 # delete workflows after 1 hour. they are still available in the archive
+        metrics:
```

Comment:
Hier sind defaults gesetzt?
Warum wird dann in den anderen Workflows auch etwas gesetzt?